discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -64,21 +58,17 @@ def _terraform_params(self, **kwargs):
             "libvirt_dns_records": kwargs.get("dns_records", {}),
             # TODO change to namespace index
             "libvirt_network_if": self.config.net_asset.libvirt_network_if,
-            "libvirt_worker_disk": operators_utils.resource_param(
-                kwargs.get("worker_disk", 21474836480), consts.OperatorResource.WORKER_DISK_KEY, operators),
-            "libvirt_master_disk": operators_utils.resource_param(
-                kwargs.get("master_disk", 128849018880), consts.OperatorResource.MASTER_DISK_KEY, operators),
+            "libvirt_worker_disk": kwargs.get("worker_disk", 26843545601),
```

Comment:
Why is `worker_disk`, and `master_disk` has different value than the default value we have on `consts.resources`?

Reply:
I restored old code, haven't checked what the default value is.